docs/conf.py

```diff
@@ -96,8 +97,12 @@
     # "navbar_align": "left",  # [left, content, right] For testing that the navbar items align properly
     # "navbar_start": ["navbar-logo", "navbar-version"],
     # "navbar_center": ["navbar-nav", "navbar-version"],  # Just for testing
-    # "navbar_end": ["navbar-icon-links", "navbar-version"]  # Just for testing
+    "navbar_end": ["version-switcher", "navbar-icon-links"],
     # "footer_items": ["copyright", "sphinx-version", ""]
+    "switcher_json_url": "/_static/switcher.json",
+    # "switcher_json_url": "https://pydata-sphinx-theme.readthedocs.io/en/latest/_static/switcher.json",
```

Comment:
This is what should be the final change before merging:

```suggestion
    "switcher_json_url": "https://pydata-sphinx-theme.readthedocs.io/en/latest/_static/switcher.json",
```